ansible/roles/cassandra-db-update/templates/data.cql.j2

```diff
@@ -52,7 +52,6 @@ ALTER KEYSPACE {{ hierarchy_keyspace_name }} WITH replication = {
   'datacenter1' : 2
 };
 
-ALTER TABLE {{ content_keyspace_name }}.content_data ADD (externallink text);
```

Comment:
@krgauraw Is it ok to not run this command on a new sunbird instance is setup?

Reply:
Same query for these - https://github.com/project-sunbird/sunbird-learning-platform/pull/1338/files#diff-93de9af5fd27b925ef3920cb86b6408858d283a26a36ecf051a0e37b3fe41168L40

https://github.com/project-sunbird/sunbird-learning-platform/pull/1338/files#diff-93de9af5fd27b925ef3920cb86b6408858d283a26a36ecf051a0e37b3fe41168L42

Reply:
yes. it is not required for new installation, as i have added those fields in the main creation query. @keshavprasadms